apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/BlockchainProcessorImpl.java

```diff
@@ -626,16 +627,16 @@ private void continuedDownloadOrTryImportGenesisShard() {
             setGetMoreBlocks(true); // turn ON blockchain downloading
             return;
         }
-        // try import genesis / shard data
-        setGetMoreBlocks(false); // turn off automatic blockchain downloading
+        // NEW START-UP logic, try import genesis OR start downloading shard zip data
+        suspendBlockchainDownloading(); // turn off automatic blockchain downloading
         log.warn("NODE IS WAITING FOR no/shard decision and proceeding with necessary data by ShardPresentEventType....");
 
-//        FileDownloader downloader; ???
-//        downloader.startDownload(id); ???
-//        FileDownloadDecision decision = downloader.prepareForDownloading(); ???
-//        FileDownloadInfo fdi = downloader.getDownloadInfo(); ??
+        ShardDownloader shardDownloader = CDI.current().select(ShardDownloader.class).get();
```

Comment:
Make it injectable or move to separate component

Reply:
Done